package.json

```diff
@@ -30,7 +30,9 @@
     "moduleNameMapper": {
       "^js(.*)$": "<rootDir>/src/js$1",
       "^dojo(.*)$": "<rootDir>/__mocks__/arcgis.js",
-      "^esri(.*)$": "<rootDir>/__mocks__/arcgis.js",
+      "esri/request": "<rootDir>/__mocks__/MapView.js",
+      "esri/views/MapView": "<rootDir>/__mocks__/MapView.js",
+      "esri/Map": "<rootDir>/__mocks__/Map.js",
       "^dijit(.*)$": "<rootDir>/__mocks__/arcgis.js",
```

Comment:
No worries, it might not be possible to have a fallback mapping. Yea that could be nice, but that can always addressed in a later commit since I'm still learning what we can do with Jest's configuration. What you have works good so I'm cool with that.